.github/workflows/release.yml

```diff
@@ -1,10 +1,6 @@
 name: Build umi-transfer
 
 on:
-  pull_request:
-    branches:
-      - main
```

Comment:
We don't want this?

Reply:
I believe, it is unnecessary. 

This action builds the binary for the different architectures. Initially, the intention of triggering it by pull requests to master was that to test the compilation prior to making a new release automatically. 

However, in practice, we are releasing that seldomly, that we can manually dispatch the workflow for the `dev` branch at suitable time points?